fe/fe-core/src/main/java/com/starrocks/catalog/OlapTable.java

```diff
@@ -1239,8 +1239,12 @@ public int getSignature(int signatureVersion, List<String> partNames) {
             LOG.debug("signature. index name: {}", indexName);
             MaterializedIndexMeta indexMeta = indexIdToMeta.get(indexId);
             // schema hash
-            adler32.update(indexMeta.getSchemaHash());
-            LOG.debug("signature. index schema hash: {}", indexMeta.getSchemaHash());
+            // schema hash will change after finish schema change. It is make no sense
+            // that check the schema hash here when doing restore
+            if (!isRestore) {
```

Comment:
Under which circumstances, the schema change will happen during the backup and restore?